requestor/worldstate.go

```diff
@@ -1,14 +1,19 @@
 package requestor
 
 import (
-	"../colorformat"
 	"fmt"
-	"github.com/tidwall/gjson"
 	"io/ioutil"
 	"net/http"
+	"time"
+
+	"github.com/t0xic0der/prudence/colorformat"
```

Comment:
Here too. Import `colorformat` like it was done before. Do let me know if you face issues while doing so.